frontends/p4/toP4/toP4.cpp

```diff
@@ -165,8 +165,10 @@ bool ToP4::preorder(const IR::P4Program *program) {
                 if (sourceFile.startsWith(p4includePath)) {
                     const char *p = sourceFile.c_str() + strlen(p4includePath);
                     if (*p == '/') p++;
+                    // TODO: This is v1model-specific code. This should be not part of the core
```

Comment:
With #4899 we can likely get rid of this code and move it into the bmv2 back end.